apps/web/app/future/[user]/[type]/embed/page.tsx

```diff
@@ -1,10 +0,0 @@
-import withEmbedSsrAppDir from "app/WithEmbedSSR";
```

Comment:
/future directory is no longer needed. We already have open PRs for migrating embeds.